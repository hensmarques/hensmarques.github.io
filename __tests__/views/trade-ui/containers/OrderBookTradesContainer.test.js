import React from 'react';
import OrderBookTradesContainer from '../../../../src/views/trade-ui/containers/OrderBookTradesContainer';

describe('<OrderBookTradesContainer/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<OrderBookTradesContainer />);
    expect(wrapper.state().classNames).toEqual(['order-book', 'recent-trades']);
    expect(wrapper.state().tabs).toEqual(['Order Book', 'Recent Trades']);
    expect(wrapper).toMatchSnapshot();
  });
});
