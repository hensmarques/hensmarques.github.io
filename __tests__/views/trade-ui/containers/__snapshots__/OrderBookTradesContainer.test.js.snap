// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderBookTradesContainer/> renders and matches the snapshot 1`] = `
<WidgetWrapper
  classNames={
    Array [
      "order-book",
      "recent-trades",
    ]
  }
  tabs={
    Array [
      "Order Book",
      "Recent Trades",
    ]
  }
>
  <Connect(OrderBook) />
  <Connect(RecentTrades) />
</WidgetWrapper>
`;
